packaging/windows/build.ps1

```diff
@@ -0,0 +1,16 @@
+# Run the build
+
+#Requires -Version 4.0
+
+$ErrorActionPreference = "Stop"
+
+. "$PSScriptRoot\functions.ps1"
+
+$msysbash = Get-MSYS2Bash "$msysprefix"
+$env:CHERE_INVOKING = 'yes'
+
+& $msysbash -l "$PSScriptRoot\compile-on-windows.sh"
```

Comment:
This probably needs to change to `compile-on-windows.sh package` to avoid a 1067 error on startup.

Reply:
If we need to have the BUILD_FOR_PACKAGING option set for things to actually work, then that should be the default, not something that needs a explicit opt-in. This is the exact type of thing I was talking about when I said yesterday that we still needed a lot of work on the Windows build infrastructure to make it production quality.

Reply:
Or, looking at this alternatively, maybe the code that we’re controlling with this should be keyed off of the target platform, not a largely unrelated configuration option that was just intended to modify how the _build system_ behaves and not how the compiled binary behaves.